java/ql/src/semmle/code/java/security/UnsafeDeserialization.qll

```diff
@@ -109,6 +116,150 @@ class SafeKryo extends DataFlow2::Configuration {
   }
 }
 
+/**
+ * A class that overrides the `android.os.Parcelable.Creator.createFromParcel` method
+ * and has the Gson call.
```

Comment:
> and has the Gson call

That might be misleading, since this method on its own does not seem to have anything to do with Gson. You are only referencing it later for dataflow tracking for Gson.

Reply:
Changed